problem.py

```diff
@@ -170,9 +170,19 @@ def get_cv(X, y):
 
 def _read_data(path, dir_name):
     X_df = pd.read_csv(os.path.join(DATA_HOME, dir_name, 'X.csv.gz'))
-    y_sparse = sparse.load_npz(
+    y = sparse.load_npz(
         os.path.join(DATA_HOME, dir_name, 'target.npz')).toarray()
-    return X_df, y_sparse
+    test = os.getenv('RAMP_TEST_MODE', 0)
+    if test:
```

Comment:
Cool. Thanks. Maybe add a small note beforehand saying that we are from now on working on limited data and that they should use the full data for their submissions

Reply:
This is only case when you do `ramp-test --quick-test`

Reply:
Then maybe it's good to also inform them that they can also use this command?